src/components/space-list/space-list.vue

```diff
@@ -53,6 +57,10 @@ export default {
   -webkit-overflow-scrolling: touch;
 }
 
+.space-list__header {
+  padding-bottom: var(--spacing-default);
+}
+
```

Comment:
Font-size looks a bit big compared to the surrounding content. Maybe add `font-size: var(--font-size-default);`?